bgpd/bgp_bfd.c

```diff
@@ -509,9 +509,13 @@ void bgp_bfd_peer_config_write(struct vty *vty, struct peer *peer, char *addr)
 	bfd_info = (struct bfd_info *)peer->bfd_info;
 
 	if (CHECK_FLAG(bfd_info->flags, BFD_FLAG_PARAM_CFG))
+#if HAVE_BFDD > 0
```

Comment:
If we are doing this, we should make these commands DEFUN_HIDDEN too.

Reply:
Makes sense. Let's help people avoid bad new practices: BFD association goes into client daemon, but BFD peer configuration goes into `bfd` configuration node.

I just pushed a commit to fix this.